gauth.go

```diff
@@ -17,24 +18,45 @@ import (
 
 func main() {
 	accountName := ""
-	isBareCode := false
+	argument := ""
 
 	if len(os.Args) > 1 {
 		accountName = os.Args[1]
```

Comment:
Hmmm might be time to move to a flags package to support arbitrary argument order…